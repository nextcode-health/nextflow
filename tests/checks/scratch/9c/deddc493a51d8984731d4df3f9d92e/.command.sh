#!/bin/bash -ue
echo alpha memry: 5 GB
echo alpha queue: all
