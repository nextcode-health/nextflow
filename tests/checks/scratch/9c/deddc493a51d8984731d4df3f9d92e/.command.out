alpha memry: 5 GB
alpha queue: all
